client/components/Account/AddCredentialForm.vue

```diff
@@ -0,0 +1,29 @@
+<!-- Form for adding a credential to an account (block style) -->
+
+<script lang="ts">
+import BlockForm from '@/components/common/BlockForm.vue';
+
+export default {
+  name: 'AddCredentialForm',
+  mixins: [BlockForm],
+  data() {
+    return {
+      url: '/api/accounts/credentials',
+      method: 'POST',
+      hasBody: true,
+      setAccount: true,
+      fields: [
+        {id: 'username', label: 'Username', value: ''},
+        {id: 'password', label: 'Password', value: ''}
+      ],
+      title: 'Add user to shared account',
```

Comment:
I think the concept of adding makes it seem as if the user account must already exist. I think to make it a bit easier to understand, it makes sense to say "Create a user to add the the shared account." or something along those lines